_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,433 @@
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Click the round button, for a surprise</p>
+		</div>
+		<div class="col">
+			<button onclick="alert('Surprise!')">Round button</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Passed Example 6
+
+This paragraph includes the [visual reference word][] "triangle" which is included in the heading of the menu, thus matching the (**visible words**) condition. The fact that the described content is on another page of the same website does not restrict this rule.
+
+```html
+<p>
+	On the <a href="/test-assets/SC1.3.3-triangle-menu-with-heading.html">information page</a> you can find more examples
+	within the triangle menu.
+</p>
+```
+
+#### Passed Example 7
+
+This paragraph includes the [visual reference words][] "wide" and "narrow". The images are indicated by these words which are also included in the [accessible names][accessible name] of the images, thus matching the (**accessible words**) condition.
+
+```html
+<p>Look at the wide image which is awesome. Now compare it with the narrow image which isn't.</p>
+<img scr="/test-assets/images/awesome_wide.jfif" alt="Wide photo of an awesome landscape." />
+<img scr="/test-assets/images/Non_awesome_narrow.jpg" alt="Narrow photo of a dull landscape." />
+```
+
+#### Passed Example 8
+
+This paragraph includes the [visual reference word][] "square" but in this case it is not identifying any [web content][], thus matching the (**no indication**) condition.
+
+```html
+<p>A square is a regular quadrilateral with four equal sides and four right angles.</p>
+```
+
+#### Passed Example 9
+
+This paragraph includes the [visual reference word][] "circle" but in this case it is not an instruction about any [web content][], thus matching the (**no instruction**) condition.
+
+```html
+<p>
+	This circle is nice.
+	<span
+		style="height: 25px;
+		     width: 25px;
+		     background-color: #bbb;
+		     border-radius: 50%;
+		     display: inline-block;"
+	>
+	</span>
+</p>
+```
+
+#### Passed Example 10
+
+This paragraph includes the [visual reference word][] "star" but there is also a heading "examples" that can be referenced, thus matching the (**non-visual references**) condition. The fact that the identified content is in an `iframe` does not restrict this rule.
+
+```html
+<p>More examples can be found when you look underneath the star or you can search for the "Examples" heading</p>
+<iframe title="star" src="/test-assets/SC1.3.3-star-with-heading.html"></iframe>
+```
+
+#### Passed Example 11
+
+This paragraph includes the [visual reference word][] "right". The content in the second column is identified with the word "right" but also identified by referencing the word "howdy", thus matching the (**non-visual reference**) condition. Note that this example is applicable because, despite the fact that the text is not visible, it is included in the accessibility tree.
```

Comment:
```suggestion
This paragraph includes the [visual reference word][] "right". The content in the second column is identified with the word "right" but also identified by referencing the word "howdy", thus matching the (**non-visual reference**) condition. Note that, despite the fact that the text is not visible, this example is applicable because it is included in the accessibility tree.
```